### PR TITLE
Olivia's Hungry-Weirdo

### PR DESCRIPTION
**Pull request by secretsharer**
# API Muncher
Congratulations! You're submitting your assignment!

## Comprehension Questions
| Question | Answer    |
| :------------- | :------------- |
| How did you go about exploring the Edamam API, how did you try querying the API? |  |
| Describe your API Wrapper.  How did you decide on the methods you created? |  |
| Describe an edge case or failure case test you wrote for your API Wrapper. |  |
| Explain how VCR aids in testing an API. |  |
| What is the Heroku URL of your deployed application? |  |
| Do you have any recommendations on how we could improve this project for the next cohort? |  |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
```
Comment (CheezItMan) [inline .gitignore:7]: You'll need to add `.env` to the `.gitignore` file.

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+group :development do
+  gem 'better_errors'
+  gem 'pry-rails'
+  gem 'foundation-rails'
```
Comment (CheezItMan) [inline Gemfile]: This gem should be outside the `development` group.

**Review by CheezItMan — CHANGES_REQUESTED** [review]
Just a few adjustments to make.
